test/function/bare-import-comment/bar.js

```diff
@@ -0,0 +1,2 @@
+// Great module
```

Comment:
Nice one! Took me some time to figure out what was going on here so I'll just write it down for future reference:
* The CJS plugin is first doing a quick regexp check to see if a module could be CJS. This check is just looking for keywords such as `module`
* If the check matches, the `transform` function repeats this test using an actual AST
* Modules that are not CJS specific were treated differently if the first or if the second check was successful. This is fixed by this PR.